serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -203,7 +203,105 @@ public class MyInterfaceOrClass {
 Avoid using `java` functions to call the external services, instead, you can use the xref:service-orchestration/orchestration-of-openapi-based-services.adoc[services orchestration features].
 ====
 
-== Custom function types
+== Camel Custom Function
+
+Kogito supports the link:{camel_url}[Camel Routes] functions within an Apache Maven project, in which you define your workflow service.
+
+[TIP]
+====
+This section brefly exemplify how to define and use Camel Routes within your workflow application. For a more in-depth guide, please see the xref:integrations/camel-routes-integration.adoc[Integrating with Camel Routes].
```

Comment:
Instead of TIP, we can convert it into a NOTE.
```suggestion
NOTE: This section briefly exemplifies how to define and use Camel Routes within your workflow application. For more information, see the xref:integrations/camel-routes-integration.adoc[Integrating with Camel Routes].
```